### ISSUE TITLE
[imports/vehicles] Modifying EDM LC500 handling/performance characteristics

### ISSUE THREAD

**Issue by definitelynothim** [InZidiuZ/legacy-assets-issue-tracker#511]
**Name:**
Me.
Baba#9198

**Summary:**
Not many people own this car (probably for a reason?), but I purchased it recently and noticed it has a lot of cons with no pros compared to other cars in it's class/price range. Even below it's price range, there are so many better options that completely outperform it in every way. For being a $150k performance car, it's lacking in all areas. This is most noticeable in any street race event.

It would be fine if there were some tradeoffs to the car that made it competitive. If the car had great handling, but ok acceleration, that would be fine. But right now, it is impossible to handle and almost never out accelerates another car in a race.

Problems the car has:

- Acceleration is OK at best compared to other performance cars.
- All levels of suspension mods at the customs shop have the same problem of throwing the car in an uncontrollable spin over any kind of bump around a corner. You have to slow down completely to get through these, making it extremely difficult to race.
- It turns extremely wide considering the above points, making it even worse to race with.
- Top speed is about 165ish without the use of bumps or downhill. Again, making the above points even worse.
- Trying to do a quick u-turn with the car feels really sluggish as it wants to completely brake through the front in order to start doing a burnout u-turn. An example of a car that doesn't wait for 0-3mph to do this is the edm corvette.

To sum up, it has nothing going for it except that it looks great. It's extremely unrewarding/frustrating to drive hard.

If I were to nitpick anything else as well, an actual LC500 has a v8 which is closer in sounding to the edm mustang than to whatever it has right now.

**Link:**
Examples of sound:
https://youtu.be/iMK8mDrIehE?t=56

**Reason:**
Anybody who buys this car is going to be frustrated when they try to street race or drive fast around the city/mountains/etc. It should at least have some pros/cons to it's characteristics.

**Comment by Nuziatella** [comment]
@joevapes3 can take a look at this

**Comment by joevapes3** [comment]
I added this to my trello and will update this post after I've taken a look at it.

**Comment by joevapes3** [comment]
LC-500 was definitely underperforming for it's class, and now the handing has been balanced to the latest meta. The turning issue you described has more to do with the vehicle being rwd but the cars suspension was a bit off so adjustments were made. Should feel alot better now. Update has been pushed on my end and will go live once the next asset merge is complete. So keep an eye out for those changes. Closing issue.
